internal/service/advancedclustertpf/model_create_ClusterDescription20240805.go

```diff
@@ -0,0 +1,223 @@
+package advancedclustertpf
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+	"go.mongodb.org/atlas-sdk/v20241023002/admin"
+)
+
+func NewAtlasReq(ctx context.Context, input *TFModel, diags *diag.Diagnostics) *admin.ClusterDescription20240805 {
+	acceptDataRisksAndForceReplicaSetReconfig, ok := conversion.StringPtrToTimePtr(input.AcceptDataRisksAndForceReplicaSetReconfig.ValueStringPointer())
```

Comment:
can we use `input.AcceptDataRisksAndForceReplicaSetReconfig.ValueString()` instead  of ValueStringPointer and use the current `StringToTime` ?

Reply:
The problem with ValueString() is it returns `""` for unknown values leading to a conversion error